lib/ansible/modules/files/stat.py

```diff
@@ -460,15 +448,7 @@ def main():
     follow = module.params.get('follow')
     get_mime = module.params.get('get_mime')
     get_attr = module.params.get('get_attributes')
-    get_md5 = module.params.get('get_md5')
-
-    # get_md5 will be an undocumented option in 2.9 to be removed at a later
-    # date if possible (3.0+)
```

Comment:
This comment implies that the intention was for this option to stick around longer than 2.8